test/common/user_rhmi_developer_permissions.go

```diff
@@ -207,3 +208,29 @@ func verifyRHMIDeveloperRHMIConfigPermissions(t TestingTB, openshiftClient *reso
 
 	verifyCRUDLPermissions(t, openshiftClient, expectedPermission)
 }
+
+func verifyRHMIDeveloperRHMIPermissions(t TestingTB, openshiftClient *resources.OpenshiftClient) {
+	t.Log("Verifying RHMI Developer permissions for RHMI Resource")
+
+	expectedPermission := ExpectedPermissions{
+		ExpectedCreateStatusCode: 403,
+		ExpectedReadStatusCode:   403,
+		ExpectedUpdateStatusCode: 403,
+		ExpectedDeleteStatusCode: 403,
+		ExpectedListStatusCode:   403,
+		ListPath:                 fmt.Sprintf(resources.PathListRHMI, RHMIOperatorNamespace),
+		GetPath:                  fmt.Sprintf(resources.PathGetRHMI, RHMIOperatorNamespace, "test-rhmi"),
```

Comment:
Actually just realized that `GetPath` should point to the existing resource, which in case of RHOAM installation is `rhoam`
```suggestion
		GetPath:                  fmt.Sprintf(resources.PathGetRHMI, RHMIOperatorNamespace, "rhoam"),
```

Reply:
Hey Pavel, renamed these now. Let me know if all is good now